apidoc/Titanium/Contacts/Contacts.yml

```diff
@@ -540,6 +540,10 @@ properties:
       by Apple.
       Since iOS 9.0, there is a native apple issue whereby it may return null if the birthday property 
       is selected on certain device models.
+      The callback contains the selected property and a 'person' object of type <Titanium.Contacts.Person>.
+      Since iOS 9.0, there is a native apple issue whereby the person is a partially object and not all
+      properties are available by default.
```

Comment:
Since iOS 9.0, apple only returns a person object with partial information. Currently it is known to only contain these properties: ? ? ?